set.mm

```diff
@@ -517997,6 +518148,71 @@ elements in a zero ring (see ~ 0ringirng ).  (Contributed by Thierry
       WBWAYEWQWRWIXOYGXOQZXIWCWDWEXNWQWMXSWNWLCXOWTXJYHXSQXNQMWFWDWGWH $.
   $}
 
+  ${
+    $d .0. f h q $.  $d .0. g q $.  $d A f h $.  $d A g q $.  $d B g $.
+    $d E f h $.  $d E g q $.  $d F f h $.  $d F g q $.  $d G f h $.
+    $d G g q $.  $d M f $.  $d M q $.  $d O f h $.  $d O g q $.  $d P g q $.
+    $d P h $.  $d Z f h $.  $d Z g $.  $d f h ph $.  $d g ph q $.
+    irredminply.o $e |- O = ( E evalSub1 F ) $.
+    irredminply.p $e |- P = ( Poly1 ` ( E |`s F ) ) $.
+    irredminply.b $e |- B = ( Base ` E ) $.
+    irredminply.e $e |- ( ph -> E e. Field ) $.
+    irredminply.f $e |- ( ph -> F e. ( SubDRing ` E ) ) $.
+    irredminply.a $e |- ( ph -> A e. B ) $.
+    irredminply.0 $e |- .0. = ( 0g ` E ) $.
+    irredminply.m $e |- M = ( E minPoly F ) $.
+    irredminply.z $e |- Z = ( 0g ` P ) $.
+    irredminply.1 $e |- ( ph -> ( ( O ` G ) ` A ) = .0. ) $.
+    irredminply.2 $e |- ( ph -> G e. ( Irred ` P ) ) $.
+    irredminply.3 $e |- ( ph -> G e. ( Monic1p ` ( E |`s F ) ) ) $.
+    $( An irreductible, monic, annihilating polynomial is the minimal
```

Comment:
irreductible -> irreducible (I think irreductible is French...)

Reply:
I did the same mistake in #4730 and obviously I can't learn from them!
Fixed in https://github.com/metamath/set.mm/pull/4788/commits/e258c749150df69ed095ab2de998ecef5b2ab046.

Reply:
It's there, in the commit mentioned.

Reply:
OK, must have been something with my browser's cache...